src/sidebar/util/test/thread-sorting-test.js

```diff
@@ -0,0 +1,125 @@
+import { sortFns } from '../thread-sorting';
+import { $imports } from '../thread-sorting';
+
+describe('sidebar/util/thread-sorting', () => {
+  let fakeRootAnnotations;
+  let fakeLocation;
+
+  beforeEach(() => {
+    // The thread argument passed to `Newest` or `Oldest` sorting functions
+    // gets wrapped with an additional Array by `*RootAnnotationDate` before
+    // being passed on to `rootAnnotations`. This unwraps that extra array
+    // and returns the original first argument to `*RootAnnotationDate`
+    fakeRootAnnotations = sinon.stub().callsFake((...args) => args[0][0]);
+    // Return the `location` property of the first argument
+    fakeLocation = sinon.stub().callsFake((...args) => args[0].location);
```

Comment:
```suggestion
    fakeLocation = sinon.stub().callsFake(annotation => annotation.location);
```

This makes it more obvious what is going on at a glance and the comment becomes superfluous.